app/models/goal.py

```diff
@@ -3,3 +3,23 @@
 
 class Goal(db.Model):
     goal_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    tasks = db.relationship("Task", back_populates="goal")
+
+    def to_dict(self):
+        return {
+            "id": self.goal_id,
+            "title": self.title
+        }
```

Comment:
👍  Good work setting up the `Goal` model. 

It wasn't mentioned in the README but whenever we're designing models/tables we should also consider whether each column should allow nullable values. In this case, is storing a goal with an empty title useful to have in our database? Or should we require every goal to have a title?  If the latter, we can add `nullable=False` to the arguments of `db.Column`. Otherwise, when that argument is omitted SQLAlchemy will set the column to `nullable=True` by default.

Also nice helper function!